sql/insert.go

```diff
@@ -98,9 +98,11 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 
 	marshalled := make([]interface{}, len(cols))
 
-	var b client.Batch
+	b := client.Batch{}
+	result := &valuesNode{}
 	for rows.Next() {
 		rowVals := rows.Values()
+		result.rows = append(result.rows, rowVals)
```

Comment:
That won't work for `RETURNING` which will have to render the results (you can return a single column or an expression such as `k+1`). I'd just do the simpler thing for now.